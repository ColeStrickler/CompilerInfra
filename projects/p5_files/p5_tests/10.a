b:bool;

g : () -> bool {
	return b == 6;
}
