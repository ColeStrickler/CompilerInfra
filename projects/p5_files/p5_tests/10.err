FATAL [4,9]-[4,15]: Invalid equality operation
Type Analysis Failed
