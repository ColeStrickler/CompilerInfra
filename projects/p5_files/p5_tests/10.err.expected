FATAL [4,9]-[4,10]: Relational operator applied to non-numeric operand
Type Analysis Failed
